CHANGELOG.md

```diff
@@ -4,6 +4,215 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.9.0] - 2023-01-20
+
+## 🚀 Features
+
+### Add support for `base64::encode()` / `base64::decode()` in Rhai ([Issue #2025](https://github.com/apollographql/router/issues/2025))
+
+Two new functions, `base64::encode()` and `base64::decode()`, have been added to the capabilities available within Rhai scripts to Base64-encode or Base64-decode strings, respectively.
+
+By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/2394
+
+### Override the root TLS certificate list for subgraph requests ([Issue #1503](https://github.com/apollographql/router/issues/1503))
+
+In some cases, users need to use self-signed certificates or use a custom certificate authority (CA) when communicating with subgraphs.
+
+It is now possible to consigure these certificate-related details using configuration for either specific subgraphs or all subgraphs, as follows:
+
+```yaml
+tls:
+  subgraph:
+    all:
+      certificate_authorities: ""
+    # Use a separate certificate for the `products` subgraph.
+    subgraphs:
+      products:
+        certificate_authorities: ""
```

Comment:
```suggestion
tls:
  subgraph:
    all:
      certificate_authorities: "${file./path/to/ca.crt}"
    # Use a separate certificate for the `products` subgraph.
    subgraphs:
      products:
        certificate_authorities: "${file./path/to/product_ca.crt}"
```

we were encouraging people to use the syntax to embed the file directly

Reply:
Hmm, I'm not sure where that went.  It's actually surprising that it disappeared and it was definitely unintentional, but this might actually be a result of doing search and replaces on the regex using unescaped an regex `replace`